R/estimators.emdi.R

```diff
@@ -81,10 +81,10 @@ estimators <- function(object, indicator, MSE, CV, ...) UseMethod("estimators")
 #'   threshold = 11064.82, transformation = "box.cox",
 #'   L = 50, MSE = TRUE, B = 50, custom_indicator =
 #'     list(
-#'       my_max = function(y, threshold) {
+#'       my_max = function(y, weights, threshold) {
```

Comment:
Also here, its the population weights?

Reply:
Ja, das ist so, wie du es sagst. Ich weiß nicht, wie ich das programmiertechnisch sonst umsetzten soll. Analog ist es ja auch so implementiert für den custom_indicator beim direct. Das heißt wir haben in diese Richtung Konistenz.